lib/releaselib.sh

```diff
@@ -751,14 +751,28 @@ release::docker::release () {
   # Activate credentials for the k8s.production.user@gmail.com
   [[ "$registry" == "gcr.io/google_containers" ]] \
    && logrun $GCLOUD config set account k8s.production.user@gmail.com
+
+  if [[ -f $(which manifest-tool) ]]; then
+    MANIFEST_TOOL=$(which manifest-tool)
+  else
+    curl -sSL https://github.com/luxas/manifest-tool/releases/download/v0.3.0/manifest-tool > ./manifest-tool
+    chmod +x ./manifest-tool
+    MANIFEST_TOOL=$(pwd)/manifest-tool
+  fi
+
+  # "docker login" to the gcr.io registry so we can push manifest lists to it
+  [[ "$registry" =~ gcr.io/ ]] && docker login -u oauth2accesstoken -p "$(gcloud auth print-access-token)" https://gcr.io
```

Comment:
would `gcloud docker -authorize-only` achieve the same thing in a slightly more-readable way?